docs/api/en/constants/BufferAttributeUsage.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<base href="../../../" />
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		<h1>Buffer Attribute Usage Constants</h1>
+
+		<p>
+			The usage constants can be used to provide a hint to the API regarding how the geometry buffer attribute will be used in order to optimize performance.
+		</p>
+
+		<h2>Code Example</h2>
+
+		<code>
+		const geometry = new THREE.SphereGeometry();
```

Comment:
Sorry for being pedantic but does this code...make sense? I mean why would somebody create an instance of `SphereGeometry` and then replace the `position` attribute?

Maybe better to use `BufferGeometry`...

Reply:
Yeah you're right BufferGeometry makes more sense with the way the snippet is written. Done!